geps/gep-713.md

```diff
@@ -100,6 +255,13 @@ The `targetRef` field MUST have the following structure:
 ```go
 // PolicyTargetReference identifies an API object to apply policy to.
 type PolicyTargetReference struct {
+    // Type is the type of the policy attachment.
+    // This can be either Direct or Hierarchical.
+    //
+    // +kubebuilder:validation:MinLength=1
+    // +kubebuilder:validation:MaxLength=253
+    Type string `json:"type"`
```

Comment:
I'd actually assumed we'd need this to be something like a standard label on the CRD. It seems like we wouldn't be able to reuse the same APIs for direct and hierarchical because the latter would also have `overrides` and `defaults` built in to the CRD.

Reply:
The idea here is to make the `TargetRef` struct be _the_ thing that defines a Policy object, and that way having this field in here lets tooling duck-type a Policy object and make some assumptions about it.